addresspoints/src/main/scala/grasshopper/addresspoints/AddressPointService.scala

```diff
@@ -1,6 +1,6 @@
 package grasshopper.addresspoints
 
-import addresspoints.api.Service
+import grasshopper.addresspoints.api.Service
```

Comment:
Most Scala projects don't follow the reverse URL namespace convention as Java projects do